src/SDammann.WebApi.Versioning/RouteVersionedControllerSelector.cs

```diff
@@ -57,15 +53,6 @@ public sealed class RouteVersionedControllerSelector : VersionedControllerSelect
             }
 
             return new ControllerIdentification(controllerName, apiVersion);
-        }
-
-        private string getControllerNameFromSubRouteData(IHttpRouteData pRouteData)

```

Comment:
Actually I do use PascalCase for method names. It's a removed code where there was camelCase.